src/main/java/org/sagebionetworks/web/server/servlet/filter/HtmlInjectionFilter.java

```diff
@@ -240,10 +240,13 @@ protected void doFilterInternal(
                 Collections.singletonList(searchQueryRawValue)
               );
             }
-            dataModel.put(
-              PAGE_TITLE_KEY,
-              "Searching for: " + query.getQueryTerm().get(0)
-            );
+            String queryTerm = query.getQueryTerm().get(0);
+            if (queryTerm != null && queryTerm.trim().length() > 0) {
```

Comment:
Special case.  If the query term is undefined, then let's not report the "Searching for" status.